edxval/tests/test_api.py

```diff
@@ -1386,3 +1386,30 @@ def test_video_image_urls_field_validation(self):
 
         self.assertEqual(len(exception_messages), 1)
         self.assertEqual(exception_messages.pop(), u'Must be a valid list of strings.')
+
+    def test_video_image_deletion_single(self):
+        """
+        Test video image deletion works as expected when an image is used by only the video being updated.
+        """
+        existing_image_name = self.course_video.video_image.image.name
+
+        api.update_video_image(self.edx_video_id, self.course_id, ImageFile(open(self.image_path2)), 'image.jpg')
+
+        with self.assertRaises(IOError) as file_open_exception:
+            ImageFile(open(existing_image_name))
+
+        self.assertEqual(file_open_exception.exception.strerror, u'No such file or directory')
+
+    def test_video_image_deletion_multiple(self):
+        """
+        Test video image deletion works as expected if an image is used by multiple course videos.
+        """
+        # Set and verify both course video images to have same image
+        shared_image = self.course_video.video_image.image.name = self.course_video2.video_image.image.name
+        self.course_video.video_image.save()
+        self.assertEqual(self.course_video.video_image.image.name, self.course_video2.video_image.image.name)
+
+        api.update_video_image(self.edx_video_id, self.course_id, ImageFile(open(self.image_path2)), 'image.jpg')
+
+        # Verify share image is not deleted by opening the file
+        ImageFile(open(shared_image))
```

Comment:
Also, we can get image file object and check it's None ? I don't know if this is the best way to test file exists?